### PR TITLE
Add row_height, header_height, initial_sort, equals dict and saving filter state

### PR DESCRIPTION
**Pull request by eliza-eliza**
### Added:
- `row_height` and `header_height` parameters in `AgTable.ag_table` to change the height of cells
- `default_sort` parameter in the column settings to set the default sorting
- `filter_include` and `filter_exclude` parameters in the column settings to set the default active and disable values on filter
- saving filters when updating the page
### Changed:
- `equals` parameter is set by the color dictionary as  value => color and renamed to `color_map`
### Removed:
- `equals_color` parameter

### Pull request checklist

- [x] Did you bump the project version?
- [x] Did you add a description to the Pull Request?
- [x] Did you add new tests?
- [x] Did you add reviewers?

### CODE REVIEW

File: src/column/number_column.jl [hunk 1]
```diff
@@ -54,22 +54,23 @@ Type defining a numeric column and its display settings.
 | `header_name::String` | `field_name` | Override `field_name` as default with custom value. |
 | `visible::Bool` | `true` | Column visibility. |
 | `filter::Bool` | `false` | Enables a filter for column values. |
+| `default_sort::AG_SORT_MODES` | `nothing` (`AG_ASC`, `AG_DESC`) | Initial sorting in the column. |
```
Comment (artememelin) [inline src/column/number_column.jl:57]:
```suggestion
| `default_sort::AG_SORT_MODES` | `nothing` (`AG_ASC`, `AG_DESC`) | Initial sorting in the column (ascending or descending order). |
```

Comment (artememelin) [reply src/column/number_column.jl:57]: I think it's worth adding to the description of all columns

File: src/column/number_column.jl [hunk 2]
```diff
@@ -54,22 +54,23 @@ Type defining a numeric column and its display settings.
...
 | `formatter::AGFormatter` | `nothing` | Formatting of the numbers. |
 | `str_format::String` | `"%s"` | String template for formatting values. The `%s` is replaced with the value. |
 | `width::Integer` | `nothing` | Column width. |
 | `cell_background::String` | `"#fff"` | Cell background. |
 | `rect_background::String` | `"#fff"` | Background color of the rectangle. |
 | `color::String` | `"#000"` | Text color. |
 | `text_align::AG_TEXTALIGN_TYPES` | `AG_CENTER` (`AG_LEFT`, `AG_RIGHT`) | Text alignment. |
-| `equals::Vector{Number}` | `Float64[]` | The values that are highlighted in color in the cell. |
-| `equals_color::String` | `"#ca3c32"` | The color that will be used to highlight values that are equal to `equals`. |
+| `color_map::Dict{Number,String}` | `Dict{Number,String}()` | A dictionary of colors by their values. |
```
Comment (artememelin) [inline src/column/number_column.jl:65]:
```suggestion
| `color_map::Dict{Number,String}` | `Dict{Number,String}()` | Dictionary with colors of values (`value` => `color`) |
```

Maybe something like this ?

File: docs/src/pages/table.md [hunk 3]
```diff
@@ -68,15 +68,14 @@ table = ag_table(
...
     ),
     AgStringColumnDef(
         field_name = "status",
         header_name = "Status",
         filter = true,
         rect_background = "#f0efeb",
-        equals = ["In Air"],
-        equals_color = "#8ac926",
-        threshold = AGThreshold("In Air", color_up = "#999999", color_down = "#ff595e"),
+        color_map = Dict("In Air" => "#8ac926", "Canceled" => "#ff595e", "Scheduled" => "#999999"),
```
Comment (artememelin) [inline docs/src/pages/table.md:78]:
```suggestion
        color_map = Dict(
            "In Air" => "#8ac926",
            "Canceled" => "#ff595e",
            "Scheduled" => "#999999",
        ),
```

File: src/column/column.jl [hunk 4]
```diff
@@ -61,6 +65,11 @@ end
     AG_PERCENT = 2
 end
 
+@enum AG_SORT_MODES begin
+    AG_ASC = 0
+    AG_DESC = 1
+end
```
Comment (gryumov) [inline src/column/column.jl:71]:
```suggestion
@enum AG_SORT_MODES begin
    AG_ASC 
    AG_DESC
end
```

**Comment by gryumov** [comment]
@eliza-eliza, thank you for your PR; I like your revisions.

**Review by gryumov — APPROVED** [review]

**Pull request merged**